.travis.yml

```diff
@@ -156,7 +156,7 @@ install:
             git -c advice.detachedHead=false clone -q --depth=1 --single-branch --branch=v17.02 http://dpdk.org/git/dpdk dpdk
             pushd dpdk
             git log --oneline --decorate
-            make config T=${TARGET} O=${TARGET}
+            make config T=${TARGET} O=${TARGET} MACHINE_CFLAGS=""
```

Comment:
Seems a simple fix. This behavior of Travis should also be a good test of the adaptability of ODP to the target microarchitecture going forward. ODP shouldn't need such compile-time flags since the goal is to have it determine this sort of low-level detail dynamically at runtime.